routes/items.js

```diff
@@ -123,48 +77,36 @@ router.post('/add', upload.single('img'), (req, res) => {
 
 /**
  * Delete item specified by item id
+ *
+ * @param {string} itemId Object ID of the item to be deleted.
+ * @param {string} eventId Object ID of the event to be modified.
  */
-router.delete('/', (req, res) => {
-	const itemId = req.body.id;
+router.delete('/', async (req, res) => {
+	const { itemId, eventId } = req.query;
 
-	if (!itemId) {
+	if (!itemId || !eventId) {
 		res.status(400).json('Required itemId not provided in requets body.');
 	}
 
-	Item.findById(itemId)
-		.then(async (item) => {
-			const { img, _id } = item;
-			await Item.findByIdAndDelete(_id);
-			await removeOneImage(img);
-
-			console.log(`Successfully deleted item: ${item}`);
-			res.json(`Sucessfully removed item ${itemId}`);
-		})
-		.catch((err) => {
-			console.log('Error: ' + err);
-			res.status(400).json(err);
+	try {
+		const deletedItem = await Item.findByIdAndDelete(itemId);
+		await Event.findByIdAndUpdate(eventId, { $pull: { items: itemId } });
+
+		const params = {
+			Bucket: 'bruinbot-item-images',
+			Key: deletedItem.imgKey,
+		};
+		s3.deleteObject(params, (err, data) => {
```

Comment:
Oh okay so you need the key for deletion? Maybe put this into a comment in the model file